twingate/internal/provider/resource/resource.go

```diff
@@ -185,6 +192,10 @@ func resourceCreate(ctx context.Context, resourceData *schema.ResourceData, meta
 		return diag.FromErr(err)
 	}
 
+	if !resource.IsActive {
+		return diag.FromErr(ErrResourceInactiveOnCreation)
```

Comment:
Just curious, why can't the provider handle this for us and just create the resource and then set it to inactive immediately to reduce friction for the end user? Seems odd to me that the API restricts this, TBH.

Reply:
thank you for your comment, it's fixed now

so now user can set inactive flag from the beginning